cmake/CompileRocksDB.cmake

```diff
@@ -30,6 +30,8 @@ if (RocksDB_FOUND)
   ExternalProject_Get_Property(rocksdb BINARY_DIR)
   set(ROCKSDB_LIBRARIES
       ${BINARY_DIR}/librocksdb.a)
+  ExternalProject_Get_Property(rocksdb INCLUDE_DIR)
+  set (ROCKSDB_INCLUDE_DIR "${RocksDB_ROOT}/include")
```

Comment:
I wonder if this broke the manual download path?

Reply:
Yeah, these two lines should be deleted. Confusingly enough, if `find_package(RocksDB)` finds Rocks, then `ROCKSDB_INCLUDE_DIR` will be set. Otherwise, we need to explicitly set it with the lines you added below.